docs/tutorials/get-started/.rebuild.md

```diff
@@ -13,7 +13,5 @@
 
 Your formula result should now be `Howdy World!`.
 
-![type:video][rebuild]{: .screenshot}
```

Comment:
just verifying that this was intentional to move it to core.md?

Reply:
Yes!

> The switch from markdown paths to HTML paths resulted in some minor rework of some tutorials templates, which were relying on markdown paths flowing into includes, in a way that can't be done through other means.